core/federated/RTI/rti.Dockerfile

```
@@ -1,13 +1,24 @@
-# Docker file for building the image of the rti
-FROM alpine:latest
+ARG BASEIMAGE=alpine:latest 
+FROM ${BASEIMAGE} as builder
 COPY . /lingua-franca
 WORKDIR /lingua-franca/core/federated/RTI
-RUN set -ex && apk add --no-cache gcc musl-dev cmake make && \
-    mkdir container && \
-    cd container && \
-    cmake ../ && \
-    make && \
-    make install
-
-# Use ENTRYPOINT not CMD so that command-line arguments go through
-ENTRYPOINT ["RTI"]
+RUN apk add --no-cache \
+        gcc musl-dev cmake make \
+        libressl-dev
+RUN mkdir -p build
+WORKDIR /lingua-franca/core/federated/RTI/build
+RUN cmake -DAUTH=on ..
+RUN make
+RUN make install
+
+WORKDIR /lingua-franca
+
+# application stage
+FROM base-${TARGETARCH} as app
+LABEL maintainer="lf-lang"
```

I'm not sure what "maintainer" is supposed to refer to, but if its the the org on Docker Hub then it should be `lflang`.
```suggestion
LABEL maintainer="lflang"
```